### ISSUE TITLE
import multiprocessing error

### ISSUE THREAD

**Issue by lcjr86** [IronLanguages/main#1420]
Hello folks,
I'm trying to import multiprocessing library on IronPython-2.7.6.3 and this is what's happen:

> > > import multiprocessing
> > > Traceback (most recent call last):
> > >   File "<stdin>", line 1, in <module>
> > >   File "C:\TEMP\IronPython-2.7.6.3\IronPython-2.7.6.3\Lib\multiprocessing__init
> > > __.py", line 64, in <module>
> > >   File "C:\TEMP\IronPython-2.7.6.3\IronPython-2.7.6.3\Lib\multiprocessing\proces
> > > s.py", line 312, in <module>
> > >   File "C:\TEMP\IronPython-2.7.6.3\IronPython-2.7.6.3\Lib\multiprocessing\proces
> > > s.py", line 309, in **init**
> > > TypeError: expected IList[Byte], got str

**Comment by simplicbe** [comment]
Thanks for reporting this issue. We'll take a loot at it!

**Comment by simplicbe** [comment]
The problem here is the constructor of `Bytes` in `Bytes.cs` that has no constructor that takes a `string` as parameter. The call `os.urandom(32)` retruns a `string` in `process.py`: `type(os.urandom(32))`. Maybe changing this overload in `Bytes.cs` solves this problem:

```
public Bytes(CodeContext/*!*/ context, [NotNull]string/*!*/ unicode, [NotNull]string/*!*/ encoding) {
            _bytes = StringOps.encode(context, unicode, encoding, "strict").MakeByteArray();
        }
```

**Comment by slozier** [comment]
@simplicbe Maybe we should be trying to figure out the underlying issue with subclassing instead of just providing fix for this specific case. The question would then be, if `bytes("a")` works, then why does the following not work:

```
class test(bytes):
    pass
test("a")
```

**Comment by simplicbe** [comment]
@slozier the problem seems to be `BytesConversion` this is not recognized when subclassing... Currently looking into it.

**Comment by simplicbe** [comment]
It seems that the `ParameterInfo` is not containing the definition for `BytesConversion`. But it should:
`ParameterInfo.IsDefined(typeof(BytesConversionAttribute), false)` in _PythonOverloadResolver_

**Comment by simplicbe** [comment]
@slozier lcjr86  got it.  I'll create a pull request

**Comment by lcjr86** [comment]
slozier, Can I ask you one question: the bug was fixed? Many thanks

**Comment by simplicbe** [comment]
Sure, just ask. Yes, I've fixed the bug yesterday, take a look at #1421 

Edit: This may also fix a few more bugs.
